tests/unit/test_code_generation.py

```diff
@@ -494,6 +522,22 @@ def func():
     )
 
 
+def test_get_class_references():
```

Comment:
you could use `@pytest.mark.xfail` to annotate this test as expected to fail: https://docs.pytest.org/en/7.1.x/how-to/skipping.html#xfail-mark-test-functions-as-expected-to-fail

Reply:
done